src/main/target/KAKUTEF4/target.h

```diff
@@ -147,10 +147,17 @@
 #define ADC_CHANNEL_3_PIN           PC1
 
 #define USE_LED_STRIP
+#if !defined(KAKUTEF4V2TRI)
 #define WS2811_PIN                      PC8
 #define WS2811_DMA_HANDLER_IDENTIFER    DMA2_ST4_HANDLER
 #define WS2811_DMA_STREAM               DMA2_Stream4
 #define WS2811_DMA_CHANNEL              DMA_Channel_7
+#else
```

Comment:
WS2811 defines are no longer necessary

Reply:
Was able to delete the DMA defines, deleting the PIN define led to a compilation error.